grading/python3pylint/Dockerfile

```diff
@@ -0,0 +1,12 @@
+# DOCKER-VERSION 1.1.0
+
+#inherit from the base container, which have all the needed script to launch tasks
+FROM    ingi/inginious-c-base
+LABEL org.inginious.grading.name="python3pylint"
+
+#add Pylint
+RUN	yum install centos-release-scl -y && \
+	yum install rh-python35 -y && \
```

Comment:
Python 3.5 is already installed in the base container via the ``python35u`` package. I'm afraid this could break some parts of the installation. Also I simply suggest to update ``python3`` container which is just of copy of ``default`` by adding the ``pylint`` related line, and ``bc``, if this is really needed.